docs/source/dev-guide/writing-tests.md

```diff
@@ -0,0 +1,148 @@
+# Writing Tests
+
+The following guide gives an overview of writing tests for the `conda` codebase.
+This article is mostly aimed at new contributors, especially those wishing
+to submit bug fixes. To give those new to writing tests in the conda codebase
```

Comment:
```suggestion
to submit bug fixes. To give those new to writing tests in the `conda` codebase
```

Reply:
@kathatherine package manager. Not the command. So, we probably just want to leave it as is, right?